tests/functional/helper.js

```diff
@@ -318,59 +326,74 @@ module.exports = {
     }
 
     // Wait for VPN client screen to move from spinning wheel to next screen
-    await this.waitForElementProperty('VPN', 'userState', 'UserAuthenticated');
-    await this.waitForElement('postAuthenticationButton');
+    await this.waitForVPNProperty('VPN', 'userState', 'UserAuthenticated');
+    await this.waitForQuery(queries.screenPostAuthentication.BUTTON.visible());
 
     // Clean-up extra devices (otherwise test account will fill up in a
     // heartbeats)
     await this._maybeRemoveExistingDevices();
 
     if (clickOnPostAuthenticate) {
-      await this.clickOnElement('postAuthenticationButton');
+      await this.waitForQuery(queries.global.SCREEN_LOADER.ready());
+      await this.clickOnQuery(
+          queries.screenPostAuthentication.BUTTON.visible());
       await this.wait();
     }
     if (acceptTelemetry) {
-      await this.waitForElement('telemetryPolicyButton');
-      await this.clickOnElement('telemetryPolicyButton');
-      await this.waitForElement('controllerTitle');
+      await this.waitForQuery(queries.global.SCREEN_LOADER.ready());
+      await this.waitForQuery(queries.screenTelemetry.BUTTON.visible());
+
+      await this.waitForQuery(queries.global.SCREEN_LOADER.ready());
+      await this.clickOnQuery(queries.screenTelemetry.BUTTON.visible());
+
+      await this.waitForQuery(queries.global.SCREEN_LOADER.ready());
```

Comment:
i would probably make this a function in the help since its used so much, 

```suggestion
      async waitForScreenLoader(){
         await this.waitForQuery(queries.global.SCREEN_LOADER.ready());
      }
```

possibly

Reply:
nit*